datafusion/core/tests/parquet/mod.rs

```diff
@@ -632,22 +633,60 @@ fn make_names_batch(name: &str, service_name_values: Vec<&str>) -> RecordBatch {
     RecordBatch::try_new(schema, vec![Arc::new(name), Arc::new(service_name)]).unwrap()
 }
 
-/// Return record batch with i8, i16, i32, and i64 sequences with all Null values
-fn make_all_null_values() -> RecordBatch {
+/// Return record batch with i8, i16, i32, and i64 sequences with Null values
```

Comment:
I think it would help to explain the shape of this data a bit more to help readers. Something like the following (though note it would change if you change the argments as I suggest below)

```suggestion
/// Return record batch with i8, i16, i32, and i64 sequences with Null values.
///
/// The first `null_values` values are null and the following 
/// `no_null_values_end - no_null_values_start` values are non null.
```